backend/user-service/src/routes/user.routes.ts

```diff
@@ -13,5 +14,6 @@ router.post('/', handleCreateUser)
 router.put('/:id', handleUpdateProfile)
 router.get('/:id', handleGetCurrentProfile)
 router.delete('/:id', handleDeleteUser)
+router.put('/password/:id', handleUpdatePassword)
```

Comment:
/:id/password would be more suitable as password is a subresource of the user with id :id